lib/bootstrap/rspec.rb

```diff
@@ -15,6 +15,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# The following line is a workaround needed because starting from JDK 21 LinkedHashSet
+# has a package private map method that JRuby bound as a usable method.
+# It's mainly related to Gradle opening java.base module and this interfere with
+# JRuby binding of methods. Full description at https://github.com/jruby/jruby/issues/8061#issuecomment-1908807511
+# Remove when Logstash bundle a JRuby version >= 9.4.6.0.
+java.util.LinkedHashSet.remove_method(:map) rescue nil
```

Comment:
Actually now that main and 8.13 are on JRuby 9.4.6.0 we should be able to remove this.

Reply:
@jsvd yep, good spot :-) removed with 1f5f73572d7ef2c140484c6b9e75dbb8886b08ef and it's green.
Ready for a quick review